src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const link = document.querySelectorAll('.list-item__link');
+
+
+document.addEventListener('DOMContentLoaded', () => {
+  const bigImg = document.querySelector('#largeImg');
+  const images = document.querySelectorAll('.gallery__img.gallery__thumb');
+  const link = document.querySelectorAll('.list-item__link');
+
+  link.forEach((elem) => {
+    elem.addEventListener('click', (obj) => {
+      obj.preventDefault();
+
+      images.forEach((image) => {
+        image.addEventListener('click', () => {
+          bigImg.setAttribute('src', elem.getAttribute('href'));
+        });
+       });
```

Comment:
The 'click' event listener on each 'image' is nested inside the 'click' event listener of 'link'. This means that the 'bigImg' source will change every time an image is clicked, but only after a 'link' is clicked. Consider restructuring this logic if the intention is to change the 'bigImg' source directly when an image is clicked.